frontend/components/notifications/NotificationsOverview.vue

```diff
@@ -205,9 +217,12 @@ a:hover {
   display: flex;
   gap: 20px;
 }
+.lists-container-column {
+  flex-grow: 1;
```

Comment:
this should also include `flex-shrink`
```suggestion
  flex: 1;
```